blueprints/util/files/__root__/utils/__name__.coffee

```diff
@@ -1,4 +1,2 @@
-<%= camelizedModuleName %> = () ->
+export default () ->
```

Comment:
@kimroen I think you can do that with
```coffeescript
export <%= camelizedModuleName %> () -> 
```

Like we have done here: https://github.com/kimroen/ember-cli-coffeescript/pull/137/files#diff-f9c146914b83a86c6a8d484dc1d877d6R4